docs/etcd_backup.md

```diff
@@ -0,0 +1,41 @@
+# Backing up etcd
+
+Kubernetes is relying on etcd for state storage. More details about the usage
+can be found [here](https://kubernetes.io/docs/admin/etcd/).
```

Comment:
Double check which version we are running ... https://coreos.com/etcd/docs/2.3.7/index.html <- 2.3.7 page